lms/djangoapps/onboarding/forms.py

```diff
@@ -489,62 +494,80 @@ class OrganizationInfoForm(BaseOnboardingModelForm):
-                                                                 "number of organizations to improve the "
-                                                                 "effectiveness of local organizations they fund and/or"
-                                                                 " partner with to deliver programs. If you were asked "
-                                                                 "to join Philanthropy University by one of your "
-                                                                 "partners or funders and that organization appears in "
-                                                                 "this list, please select it."),
-                                         widget=forms.CheckboxSelectMultiple,
-                                         required=False,
-                                         error_messages={
-                                             'required': ugettext_noop(NO_OPTION_SELECT_ERROR.format("Partner's")),
-                                         })
+    is_org_url_exist = forms.ChoiceField(
+        label=ugettext_noop('Does your organization have a website?'),
+        choices=((1, ugettext_noop('Yes')), (0, ugettext_noop('No'))),
+        label_suffix='*',
+        widget=forms.RadioSelect,
+        initial=1,
+        help_text=ugettext_noop(
+            "If your organization does not have its own website, "
+            "but it does have a page on another platform like "
+            "Facebook or WordPress, please answer Yes here and "
+            "give the address for this page."
+        ),
+        error_messages={
+            'required': ugettext_noop('Please select an option for "Does your organization have a webpage?"')
+        }
+    )
+
+    org_type = forms.ChoiceField(
+        label=ugettext_noop('Organization Type'),
+        label_suffix='*',
+        error_messages={
+            'required': ugettext_noop(NO_OPTION_SELECT_ERROR.format('Organization Type'))
+        }
+    )
+
+    level_of_operation = forms.ChoiceField(
+        label=ugettext_noop('Level of Operation'),
+        label_suffix='*',
+        error_messages={
+            'required': ugettext_noop(NO_OPTION_SELECT_ERROR.format('Level of Operation'))
+        }
+    )
+
+    focus_area = forms.ChoiceField(
+        label=ugettext_noop('Primary Focus Area'),
+        label_suffix='*',
+        error_messages={
+            'required': ugettext_noop(NO_OPTION_SELECT_ERROR.format('Primary Focus Areas'))
+        }
+    )
+
+    total_employees = forms.ChoiceField(
+        label=ugettext_noop('Total Employees'),
+        label_suffix='*',
+        help_text="An employee is a member of your staff who is paid for their work. "
+                  "A staff member working full-time counts as 1 employee; a staff "
+                  "member working half-time counts as 0.5 of an employee. Please "
+                  "include yourself in your organization's employee count.",
```

Comment:
Can we please use single qoutes?

Reply:
Good suggestion, But the thing is that in this file all the help texts are using double quotes because single quotes are used inside double quotes like `organization's` in this case. Also this is out of scope for this ticket.